test/test_sync.cpp

```diff
@@ -6805,52 +6804,56 @@ TEST(Sync_TransformAgainstEmptyReciprocalChangeset)
         tr->commit();
     }
 
-    MultiClientServerFixture fixture(3, 1, dir, test_context);
-    fixture.start();
-
-    util::Optional<Session> seed_session = fixture.make_bound_session(0, seed_db, 0, "/test");
-    util::Optional<Session> db_1_session = fixture.make_bound_session(1, db_1, 0, "/test");
-    util::Optional<Session> db_2_session = fixture.make_bound_session(2, db_2, 0, "/test");
-
-    seed_session->wait_for_upload_complete_or_client_stopped();
-    db_1_session->wait_for_download_complete_or_client_stopped();
-    db_2_session->wait_for_download_complete_or_client_stopped();
-    seed_session.reset();
-    db_2_session.reset();
+    {
```

Comment:
What are these changes from?

Reply:
It's from a thread sanitizer issue unrelated to this pr that just surfaced now